src/RouteProvider/PropertyAccess/PurgatoryPropertyAccessor.php

```diff
@@ -24,9 +24,8 @@ public function __construct(
 
     /**
      * @param object|array<array-key, mixed> $objectOrArray
-     * @param string|PropertyPathInterface   $propertyPath
      */
```

Comment:
phpstan still requires array definition, not sure if we should just ignore it or keep phpdoc

Reply:
Let's keep it